provider-tutorials/becoming-erigon-provider.md

```diff
@@ -0,0 +1,121 @@
+# Tutorial for becoming Erigolem provider
+
+{% hint style="info" %}
+`Erigolem` is a codename for "Erigon on Golem" project. More information can be found at [Example service - managed Erigon](https://github.com/golemfactory/yagna-docs/tree/master/requestor-tutorials/service-development/service-example-2-managed-erigon.md)
+{% endhint %}
+
+Here we describe an [installation script](https://github.com/golemfactory/yagna-service-erigon/blob/master/install_provider.sh) which should make the whole process of becomming Erigolem provider fun and smooth. Script is prepared for Ubuntu 18.04 or later and should work on other Linux distributions after some modifications.
+It's not intended to be run on personal computers because
+
+* It creates a separate user which will run provider daemon,
+* It installs nginx web server and certbot,
+* Optionally, it generates Let's Encrypt certificate and sets certbot to remind you about certificate renewals.
+
+## Requirements
+
+* Ubuntu 18.04 or later
+* Root privileges
+* Public IP address or DNS name
+* Ability to open a port for the Erigon service
+
+{% hint style="info" %}
+By default nginx reverse-proxy is listening on standard Erigon port `8545`. You need to open this port for ingres traffic. If you wish your Erigolem service to be available on different port number (e.g. `8454` cannot be opened) you need to edit the installation script.
```

Comment:
```suggestion
By default nginx's reverse-proxy is listening on the standard Erigon port `8545`. You need to open this port for incoming traffic. If you wish your Erigolem service to be available on a different port number (e.g. because `8454` cannot be opened), you need to edit the installation script.
```